### DISCUSSION TITLE
WLAN Pi docs as of June 6th

### DISCUSSION THREAD

**Discussion by ThatIdiotSid** [WLAN-Pi/feedback#23] [General Inquiries] [answered]
I've noticed that the docs on the main site still have some blank areas.   I've got a day job that has me working upwards of 60+ hrs a week,   but I've been a huge fan of this project since the previous incarnation.    If I can free up some time,  I wouldn't mind volunteering to at least help out a little bit, to flesh out the docs more.

(just throwing that out there,  since I want this project to succeed)

**Comment by jolla** [comment, marked as answer]
Sorry for the late reply, if the offer still stands assistance with documentation would be welcomed and appreciated by all.
